lib/challenge.js

```diff
@@ -33,7 +42,10 @@
 var pet_owner;
 
 // Answer Starts Here
-
+ {
+ name: "Jane Doe"
+ address: "1 Toll Rd."
+ }
```

Comment:
The same missing commas apply here. This object literal is not being assigned at all, though. That means the reference is immediately lost upon creation.